libs/dao/src/main/java/com/akto/util/modifier/JWTPayloadModifier.java

```diff
@@ -0,0 +1,25 @@
+package com.akto.util.modifier;
+
+public class JWTPayloadModifier extends JwtModifier {
```

Comment:
Please fix the name of this class. It should inform something that you are trying to merge 2 tokens